packages/expo-contacts/src/next/types/Container.ts

```diff
@@ -1,21 +1,97 @@
 import { Contact } from './Contact';
 import { Group } from './Group';
 
+/**
+ * Represents the account type of a container.
+ */
 export type ContainerType = 'local' | 'exchange' | 'cardDAV' | 'unassigned' | 'unknown';
 
 export declare class Container {
+  /**
+   * Constructor for Container instance.
+   * While you can instantiate this class directly if you have an ID, the recommended way to obtain a `Container` instance
+   * is to use static methods like `Container.getAll` or `Container.getDefault`.
+   *
+   * @param id - The unique identifier of the container.
+   * @platform ios
+   */
   constructor(id: string);
+
+  /**
+   * The unique identifier for the container.
+   * @platform ios
+   */
   readonly id: string;
 
+  /**
+   * Retrieves the name of the container.
+   * @returns A promise resolving to the container name string (e.g., "iCloud", "Gmail") or `null` if not available.
+   * @platform ios
+   * @example
+   * ```ts
+   * const name = await container.getName(); // 'iCloud'
+   * ```
+   */
   getName(): Promise<string | null>;
+
+  /**
+   * Retrieves the type of the container.
+   * @returns A promise resolving to the {@link ContainerType} (e.g., 'cardDAV', 'exchange') or `null`.
```

Comment:
```suggestion
   * @returns a promise resolving to the [`ContainerType`](#containertype) (for example, 'cardDAV', 'exchange') or `null`.
```

The `@link` is not supported. Let's add internal links in markdown format.